include/RAJA/index/IndexValue.hpp

```diff
@@ -336,19 +336,15 @@ constexpr RAJA_HOST_DEVICE RAJA_INLINE
 
 namespace internal{
 template<typename FROM, typename Enable = void>
-struct StripIndexTypeT {};
+struct StripIndexTypeT {
```

Comment:
What is the second template parameter for?

Reply:
@rhornung67 it's for the enable_if on the specialization